data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<tool id="ncbi_taxonomy_fetcher" name="NCBI" tool_type="manage_data" version="1.0.0">
+    <description>taxonomy downloader</description>
+    <stdio>
+        <exit_code description="Error" level="fatal" range="1:" />
+    </stdio>
+    <command>
+    <![CDATA[
+        curl -o taxdump.tar.gz ${taxonomy_url} && tar xzf taxdump.tar.gz && python $__tool_directory__/data_manager.py --name "${database_name}" --out "${out_file}"
+    ]]>
```

Comment:
`curl` is not installed on every system, so I think we need to depend on https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_libcurl_7_35/tool_dependencies.xml